modules/ts/misc/perf_tests_timing.py

```diff
@@ -78,7 +100,7 @@
             suit_time = 0
             suit_num = 0
             fails_num = 0
-            for name in sorted(test_cases.iterkeys(), key=alphanum_keyselector):
+            for name in sorted(test_cases, key=alphanum_keyselector):
```

Comment:
With Python 2 it does not crash, but produce broken report. For 3.4 it's important to presume 2.7 support.
```
python2 ../opencv-3.4/modules/ts/misc/perf_tests_timing.py ./imgproc_perf.xml          

Overall time: 0.79 min

Module  Testsuit Time (min) Num of tests Failed
        r                                      
        e                                      
        s                                      
        i                                      
        z                                      
        e                                      
        D                                      
        o                                      
        w                                      
        n                                      
        L                                      
        i                                      
        n                                      
        e                                      
        a                                      
        r                                      
        N                                      
        o                                      
        n                                      
        E                                      
imgproc x          0.304         60            
        a                                      
        c                                      
        t                                      
        :                                      
        :                                      
        M                                      
        a                                      
        t                                      
        I                                      
        n                                      
        f                                      
        o                                      
        _                                      
        S                                      
        i                                      
        z                                      
        e                                      
        P                                      
        a                                      
        i                                      
        r                                      
        r                                      
        e                                      
        s                                      
        i                                      
        z                                      
        e                                      
        U                                      
        p                                      
        L                                      
        i                                      
        n                                      
        e                                      
        a                                      
        r                                      
        N                                      
        o                                      
        n                                      
        E                                      
        x                                      
imgproc a          0.285         24            
        c                                      
        t                                      
        :                                      
        :                                      
        M                                      
        a                                      
        t                                      
        I                                      
        n                                      
        f                                      
        o                                      
        _                                      
        S                                      
        i                                      
        z                                      
        e                                      
        P                                      
        a                                      
        i                                      
        r                                      
        r                                      
        e                                      
        s                                      
        i                                      
        z                                      
        e                                      
        D                                      
        o                                      
        w                                      
        n                                      
        L                                      
        i                                      
        n                                      
        e                                      
        a                                      
        r                                      
        :                                      
imgproc :          0.108         20            
        M                                      
        a                                      
        t                                      
        I                                      
        n                                      
        f                                      
        o                                      
        _                                      
        S                                      
        i                                      
        z                                      
        e                                      
        _                                      
        S                                      
        i                                      
        z                                      
        e                                      
        r                                      
        e                                      
        s                                      
        i                                      
        z                                      
        e                                      
        U                                      
        p                                      
        L                                      
        i                                      
        n                                      
        e                                      
        a                                      
        r                                      
        :                                      
        :                                      
imgproc M          0.096         8             
        a                                      
        t                                      
        I                                      
        n                                      
        f                                      
        o                                      
        _                                      
        S                                      
        i                                      
        z                                      
        e                                      
        _                                      
        S                                      
        i                                      
        z                                      
        e
```

Reply:
Apparently the problem was not with `.iterkeys()`, it is the script `perf_tests_timing.py`. It does not work in _python2_ with current _3.4_ branch as well.